pkg/client/cmd/app.go

```diff
@@ -458,6 +541,10 @@ func init() {
 	// App logs
 	appLogsCmd.Flags().Int64("lines", 10, "number of lines")
 	appLogsCmd.Flags().Bool("follow", false, "follow logs")
+	// App autoscale
+	appAutoScaleSetCmd.Flags().Int32("min", flagNotDefined, "Auto scale min size.")
+	appAutoScaleSetCmd.Flags().Int32("max", flagNotDefined, "Auto scale max size.")
+	appAutoScaleSetCmd.Flags().Int32("cpu-percent", flagNotDefined, "The target average CPU utilization (represented as a percent of requested CPU) over all the pods. If it's not specified or negative, the current autoscaling policy will be used.")
```

Comment:
`If it's not specified or negative, the default kubernetes autoscaling policy will be used`